babel-preset/configure.js

```diff
@@ -70,6 +74,11 @@ module.exports = function(options) {
       require('babel-plugin-transform-es3-property-literals'),
       require('babel-plugin-transform-flow-strip-types'),
       require('babel-plugin-transform-object-rest-spread'),
+      require('babel-plugin-transform-react-display-name'),
+      require('babel-plugin-transform-react-jsx'),
+      require('babel-plugin-transform-react-jsx-source'),
```

Comment:
I actually think we don't want this. It will embed the full file path which means we'd be shipping `var _jsxFileName = '/Users/poshannessy/FB/code/draft-js/src/component/contents/DraftEditorContents.react.js'` to npm, which is pretty clowny. We could potentially make it a dev-only or opt-in transform but I'm inclined to just drop it. If consumers want it they can specify it themselves.